ovro_alert/buffer_trigger.py

```diff
@@ -0,0 +1,14 @@
+from time import sleep
+from astropy.time import Time
+from ovro_alert.lwa_alert_client import LWAAlertClient
+from mnc import control
+
+
+xhosts = ['lxdlwagpu02']
+con = control.Controller('proj/lwa-shell/mnc_python/config/lwa_config_calim.yaml', xhosts=xhosts)
+pipelines = con.pipelines
+p = pipelines[0]
```

Comment:
Do we actually need to send a trigger command to all pipelines? And each would get a different `dump_path`?

Reply:
you do need to send a trigger to each (though I suppose this could be changed so that all pipelines listen to the same etcd key). Once all the nodes are properly set up, I would imagine half the pipelines should dump to `/data0` and the other half to `/data1` (such that the data path respects the NUMA affiliation of the pipeline)